### PR TITLE
update font optimization page

### PR DESCRIPTION
**Pull request by ismaelrumzan**
Update to using `pages/` instead of `app/`

## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Errors have a helpful link attached, see `contributing.md`

## Feature

- [ ] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Documentation added
- [ ] Telemetry added. In case of a feature if it's used or not.
- [ ] Errors have a helpful link attached, see `contributing.md`

## Documentation / Examples

- [x] Make sure the linting passes by running `pnpm build && pnpm lint`
- [ ] The "examples guidelines" are followed from [our contributing doc](https://github.com/vercel/next.js/blob/canary/contributing/examples/adding-examples.md)

### CODE REVIEW

File: docs/basic-features/font-optimization.md [hunk 1]
```diff
@@ -26,49 +26,82 @@ Automatically self-host any Google Font. Fonts are included in the deployment an
...
+    </>
   )
 }
 ```
 
+#### Single page usage
+
+To use the font on a single page, add it to the specific page as shown below:
+
+```js:pages/index.js
```
Comment (hanneslund) [inline docs/basic-features/font-optimization.md:91]: Should the example use the font in a component that the page renders, to make it clear that you don't have to call the font function in the actual page for it to work? What do you think?
```js:pages/index.js
import Header from "../components/header.js"

export default function Home() {
  return (
    <>
        <Header>Test</Header>
        <p>Hello World</p>
    </>
  );
}
```

```js:components/header.js
import { inter } from "../styles/fonts.js"

export default function Header({ children }) {
  return <h1 className={inter.className}>{children}</h1>;
}
```

**Review by hanneslund — CHANGES_REQUESTED** [review]

File: docs/basic-features/font-optimization.md [hunk 2]
```diff
@@ -26,49 +26,82 @@ Automatically self-host any Google Font. Fonts are included in the deployment an
 
 Import the font you would like to use from `@next/font/google` as a function. We recommend using [**variable fonts**](https://fonts.google.com/variablefonts) for the best performance and flexibility.
 
-```jsx
-// app/layout.tsx
-import { Inter } from '@next/font/google'
+To use the font in all your pages, add it to `_app.js` or `_app.tsx` under `/pages` as shown below:
```
Comment (delbaoliveira) [inline docs/basic-features/font-optimization.md:29]:
```suggestion
To use the font in all your pages, add it to [`_app.js` file](https://nextjs.org/docs/advanced-features/custom-app) under `/pages` as shown below:
```

**Review by delbaoliveira — DISMISSED** [review]

File: docs/basic-features/font-optimization.md [hunk 3]
```diff
@@ -156,3 +182,4 @@ Every time you call the `localFont` or Google font function, that font is hosted
     <small>Learn how to optimize images with the Image component.</small>
   </a>
 </div>
+````
```
[the remark this answers is not in the document]
Comment (delbaoliveira) [reply docs/basic-features/font-optimization.md:185]:
```suggestion
```
```

[the remark this answers is not in the document]
Comment (delbaoliveira) [reply docs/basic-features/font-optimization.md:185]:
```suggestion
```

**Review by timneutkens — APPROVED** [review]

**Review by timneutkens — APPROVED** [review]

**Pull request merged**